src/bundle/Resources/public/js/scripts/core/multilevel.popup.menu.js

```diff
@@ -0,0 +1,385 @@
+
+        initBranch(
+            triggerElement,
+            branchElement,
+            referenceElement = null,
+            placement = 'right-start',
+            fallbackPlacements = ['right-end', 'left-start', 'left-end'],
+            processBranchAfter = () => {},
+            processBranchItemAfter = () => {},
+        ) {
+            doc.body.appendChild(branchElement);
+
+            const isTopBranch = !triggerElement.classList.contains('ibexa-popup-menu__item');
+            const branchItems = this.getBranchItems(branchElement);
+            const offset = isTopBranch ? [0, 3] : [-8, 2];
+
+            const popperInstance = Popper.createPopper(referenceElement ?? triggerElement, branchElement, {
+                placement,
+                modifiers: [
+                    {
+                        name: 'flip',
+                        enabled: true,
+                        options: {
+                            fallbackPlacements,
+                        },
+                    },
+                    {
+                        name: 'offset',
+                        options: {
+                            offset,
+                        },
+                    },
+                ],
+            });
+
+            branchElement.popperInstance = popperInstance;
+
+            if (isTopBranch) {
+                triggerElement.addEventListener('click', this.handleItemWithSubitemsClick, false);
+
+                return;
+            }
+
+            triggerElement.addEventListener(
+                'mouseenter',
+                () => {
+                    this.hoveredItemsBranches.add(branchElement);
+                    this.updateBranchAndParentBranchesOpenState(branchElement);
+                },
+                false,
+            );
+            triggerElement.addEventListener(
+                'mouseleave',
+                () =>
+                    setTimeout(() => {
+                        this.hoveredItemsBranches.delete(branchElement);
+                        this.updateBranchAndParentBranchesOpenState(branchElement);
+                    }, 50),
+                false,
+            );
+            branchElement.addEventListener(
+                'mouseenter',
+                () => {
+                    this.hoveredBranches.add(branchElement);
+                    this.updateBranchAndParentBranchesOpenState(branchElement);
+                },
+                false,
+            );
+            branchElement.addEventListener(
+                'mouseleave',
+                () => {
+                    setTimeout(() => {
+                        this.hoveredBranches.delete(branchElement);
+                        this.updateBranchAndParentBranchesOpenState(branchElement);
+                    }, 50);
+                },
+                false,
+            );
+
+            processBranchAfter(branchElement);
+            branchItems.forEach((itemElement) => processBranchItemAfter(itemElement));
```

Comment:
```suggestion
            branchItems.forEach(processBranchItemAfter);
```
nitpick

Reply:
This was intended, I didn't want `forEach` to pass extra parameters to `processBranchItemAfter`.